Cargo.toml

```diff
@@ -3,10 +3,12 @@ resolver = "2"
 members = [
   "crates/async-dropper",
   "crates/async-dropper-derive",
-  "crates/async-dropper-simple",
+  "crates/async-dropper-simple", 
+  "crates/macros_internal",
 ]
 
 [workspace.dependencies]
+async-scoped = "0.9.0"
```

Comment:
I'm not too sure that many people are chafing at the requirement to use a `'static` lifetime here -- just curious but do you have an example use case?

Reply:
See the `move` keyword where the callback is created.